CONTRIBUTING.md

```diff
@@ -21,6 +21,14 @@ phpize > /dev/null && \
 make clean > /dev/null && make all > /dev/null && make install
 ```
 
+To verify that the installation was successful, run the following command which indicates the version of the installed
+extension. It must end with `-dev` suffix.
+
+```
+php --ri mongodb
+```
+
+
```

Comment:
```suggestion
```

Not sure if the extra line was intentional, but other sections use a single break before the next title. I'm open to changing this but if so we should just do it across the entire file (same PR or a different is OK by me).